service/src/test/java/bio/terra/workspace/service/workspace/AwsCloudContextConnectedTest.java

```diff
@@ -0,0 +1,175 @@
+package bio.terra.workspace.service.workspace;
```

Comment:
So this is basically a test to be sure that WSM can read the discovery bucket with the current library it's compiled with and payload/schema data in the bucket?

Reply:
Yes, plus test the WSM functions that make a cloud context from the environment